specification.md

```diff
@@ -0,0 +1,89 @@
+# OpenTelemetry/Prometheus Compatibility Specification
+
+Status: [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md)
+
+## Abstract
+
+OpenTelemetry is aiming to provide compatibility with
+Prometheus and OpenMetrics. This document explains the
+extent of the support. OpenTelemetry collector and libraries
+will align with the compatibility requirements defined in
+this spec.
+
+## Goals
+
+* OpenTelemetry collector can be used as a drop-in replacement
+  for Prometheus server to scrape and export metrics data.
+* OpenTelemetry collector should export OLTP-compatible Prometheus
+  time series to OpenTelemetry metrics exporters.
+* OpenTelemetry libraries should implement exporters to publish
+  Prometheus metrics.
+
+## Differences and Limitations
```

Comment:
How about metadata? (help, type, etc. which is experimentally available in prometheus remote-write)

Reply:
The OTLP proto includes Help and Units. Type information is part of the data point, and we have a concept known as "Temporality" to distinguish Delta and Cumulative points. There are several open issues being discussed here, including one about how to represent the GaugeHistogram type, for example: https://github.com/open-telemetry/opentelemetry-proto/issues